.github/workflows/e2eLocal.yml

```diff
@@ -0,0 +1,94 @@
+
+name: Run end-to-end tests
+# on:
+#   workflow_dispatch: # because sometimes you just want to force a branch to have tests run
+#   pull_request:
+#     branches:
+#       - "**"
+on:
+  push:
+    branches:
+      - "**"
+jobs:
+
+  cypress-local-env:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    # If we end up in a position where our github runners are out of memory we can increase our swap space or provision private runners.
+    # I've set our swap size to 10gb because running all the apps and cypress is a bit intensive and doing this prevents random hangups
+    # https://github.com/pierotofy/set-swap-space/blob/master/action.yml
```

Comment:
I appreciate the tag. I would rather see us use our own action rather than pulling in an external one. Fewer external dependencies = fewer possible intrusion attempts for us to worry about. (That, and the repo we're citing doesn't seem to have an explicit license, which I don't want us to get in trouble for down the road.)

Reply:
I went ahead and created that action, going to resolve this.